TrafficCapture/dockerSolution/src/main/docker/opensearchDashboards/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM opensearchproject/opensearch-dashboards:2.7.0
```

Comment:
Good point -- the reason this was initially used is that it allows us to load a certificate and use ssl verification. Right now we're not actually taking advantage of that (it was causing some issues), but it may also come in handy to load index mappings and preconfigure a dashboard. If you don't feel strongly, I'd lean towards keeping it this way for now, but it's not actually giving us any benefits in the short term.